### PR TITLE
Chore/ui-fix-portal-workspace

### PR DESCRIPTION
**Pull request by Noggling**
# Description

Fix side sheet going over buttons
and table height

Completes: [AB#188110](https://dev.azure.com/Equinor/fa63ceea-8883-41e2-8823-a3b54e4be178/_workitems/edit/188110)

## Checklist:

-   [x] I have performed a self-review of my own code.
-   [x] I have linked my DevOps task using the AB# tag.
-   [x] My code is easy to read, and comments are added where needed.
-   [ ] My code is covered by tests.

### CODE REVIEW

File: src/packages/Table/Components/Table.tsx [hunk 1]
```diff
@@ -12,17 +12,21 @@ import { Table as TableWrapper, TableCell, TableRow } from './Styles';
 interface DataTableProps<TData extends TableData> {
     options: TableOptions<TData>;
     FilterComponent?: React.FC<{ filterId: string }>;
+    height?: number;
+    itemSize?: number;
 }
 
-const topBarHeight = 64;
-const itemSize = 35;
+const defaultHeight = 600;
```
Comment (jaysencpp) [inline src/packages/Table/Components/Table.tsx:19]: In sidesheetContext.ts the default sizes are named DEFAULT_HEIGHT, maybe we want to follow the same naming convention here, DEFAULT_HEIGHT and DEFAULT_ITEM_SIZE

File: src/packages/Utils/Hooks/useEventListener.ts [hunk 2]
```diff
@@ -0,0 +1,58 @@
+import { RefObject, useEffect, useRef } from 'react';
+
+/**Use EventListener with simplicity by React Hook. It takes as parameters a eventName,
+ * a call-back functions (handler) and optionally a reference element.
+ * You can see above two examples using useRef and window based event. */
```
Comment (jaysencpp) [inline src/packages/Utils/Hooks/useEventListener.ts:5]: I don't see any examples above?

**Review by jaysencpp — APPROVED** [review]
Cool hooks

File: src/packages/Table/Components/Table.tsx (hunk 1 above)
Comment (Noggling) [reply src/packages/Table/Components/Table.tsx:19]: Enig! jeg endrer ;)

File: src/packages/Utils/Hooks/useEventListener.ts (hunk 2 above)
Comment (Noggling) [reply src/packages/Utils/Hooks/useEventListener.ts:5]: hehe klasisk eksemple på copy paste :P

**Pull request merged**